docs/modules/mongodb_watchdog.md

```diff
@@ -1,15 +1,32 @@
 # Logger: `mongodb_watchdog`
 
-The module uses a separate database, using the logger database alias in settings. Do NOT point that alias to the same database as default, because the module drops the logger database when uninstalling, which would drop all your other data with it.
+The `mongodb_watchdog` module stores the watchdog entries for your Drupal site in MongoDB collection as mentioned in [Settings Configuration](../../install#settings-configuration)
 
-* mongodb.watchdog.items: the maximum item limit on the capped collection used by the module. If not defined, it defaults to 10000. The actual (size-based) limit is derived from this variable, assuming 1 kiB per watchdog entry.
+This module uses a separate database, using the `logger` database alias in settings. Do NOT point that alias to the same database as `default`, because the module drops the `logger` database when uninstalling, which would drop all your other data with it.
 
-* mongodb.watchdog.limit: the maximum severity level (0 to 7, per RFC 5424) to save into watchdog. Errors below this level (with a higher numerical level) will be ignored by the module. If not defined, all events will saved.
+ * `mongodb.watchdog.items`
+    * the maximum item limit on the capped collection used by the module.
+    * if not defined, it defaults to 10000.
+    * the actual (size-based) limit is derived from this variable, assuming 1 [kiB][kiBwiki] per watchdog entry.
 
-* mongodb.watchdog.items_per_page: the maximum number of events displayed on the event details page.
+ * `mongodb.watchdog.limit`
+    * the maximum severity level (0 to 7, per RFC 5424) to save into watchdog.
```

Comment:
Initial caps should be consistent at least within a file: either all on or all off